editor/import/3d/post_import_plugin_skeleton_rest_fixer.cpp

```diff
@@ -603,6 +609,30 @@ void PostImportPluginSkeletonRestFixer::internal_process(InternalImportCategory
 					}
 				}
 			}
+			if (p_options.has("retarget/rest_fixer/reset_all_bone_poses_after_import") && !bool(p_options["retarget/rest_fixer/reset_all_bone_poses_after_import"])) {
```

Comment:
If store the bool results of this earlier, you can avoid unnecessary calculations for some pose changes in line 117-120 and 131 as well.

Reply:
I considered this, but the number of calculations are low, and doing them in that part of the code preserves correctness (counteracting scale changes).